src/View/Helper/FormHelper.php

```diff
@@ -160,8 +161,8 @@ public function submit($caption = null, array $options = [])
             $class = array_unique(array_merge($class, $optionsClass));
             unset($options['class']);
         }
-
         $options = Hash::merge($options, ['class' => $class]);
+        $options['class'] = ButtonWidget::applyStyle($options['class']);
```

Comment:
There's one with same classname in cake too but under different namespace for different purpose :)